src/main.rs

```diff
@@ -107,7 +109,13 @@ pub fn list_languages(assets: &HighlightingAssets, term_width: usize) {
     let desired_width = term_width - longest - separator.len();
 
     for lang in languages {
-        print!("{:width$}{}", lang.name, separator, width = longest);
+        write!(
```

Comment:
We call stdout() multiple times here. It might make sense to get [a lock on stdout](https://doc.rust-lang.org/stable/std/io/struct.Stdout.html#method.lock) and just print to that handle.